test/fixtures/cdb_crankshaft/cdb_kmeans.sql

```diff
@@ -10,3 +12,38 @@ CREATE OR REPLACE FUNCTION cdb_crankshaft.CDB_KMeans(query text, no_clusters int
     RETURN;
     END;
 $$ LANGUAGE plpgsql;
+
+CREATE OR REPLACE FUNCTION cdb_crankshaft.CDB_Contour(
```

Comment:
Why not to use the same approach that we did in `gravity`?. I mean, create a separated file with `CDB_Cotour`